app/test/block_size_test.go

```diff
@@ -129,7 +129,7 @@ func (s *IntegrationTestSuite) TestMaxBlockSize() {
 			for _, hash := range hashes {
 				// TODO: once we are able to query txs that span more than two
 				// shares, we should switch to proving txs existence in the block
```

Comment:
[question] is this comment still necessary after we pass `prove=true` below?

```suggestion
```

Reply:
good catch d1f615293bfad076cda6913880450b160842c383